bin/i18n-codeorg/in.sh

```diff
@@ -20,7 +20,6 @@ mkdir -p $loc_dir
 cp_in $orig_dir/en.yml $loc_dir/base.yml
 
 # Copy in needed files from dashboard
-cp_in $orig_dir/blocks.en.yml $loc_dir/blocks.yml
```

Comment:
Moved to: https://github.com/code-dot-org/code-dot-org/pull/52622/files#diff-9b74aafab3e6e0ca501ff82f47a8c610037c21ec1cb0960c0a823055e3cd68d1R49